src/main/java/io/github/moulberry/notenoughupdates/miscfeatures/CustomItemEffects.java

```diff
@@ -346,7 +348,7 @@ public void onOverlayDrawn(RenderGameOverlayEvent.Post event) {
 						Minecraft.getMinecraft().objectMouseOver.sideHit, 1));
 				if (hover.getBlock() == Blocks.air) {
 
-					if (heldInternal != null && heldInternal.equals("BUILDERS_WAND")) {
+					if (heldInternal != null && (heldInternal.equals("BUILDERS_WAND") || heldInternal.equals("BUILDERS_RULER"))) {
```

Comment:
Since you split the entire behaviour further down anyway, why not move this out to two seperate methods with common behaviour in another utility method.